Bindings/bindings.json

```diff
@@ -7,23 +7,67 @@
         "triggerRefHelp": "You can use brackets ({value}) and percent symbols (%value%) to format your path based on parameters from the trigger object."
     },
     "bindings": [
-        {
-            "type": "timerTrigger",
-            "displayName": "Timer",
-            "direction": "trigger",
+      {
```

Comment:
Spacing is inconsistent with rest of file. I'm fine changing to 2 spaces, but then let's convert whole file